### DISCUSSION TITLE
How to scroll Animated FlatList to a specific item when the component is first rendered?

### DISCUSSION THREAD

**Discussion by mxm87** [software-mansion/react-native-reanimated#1644] [Q&A] [answered]
Please delete this, found the answer

**Comment by karol-bisztyga** [comment, marked as answer]
ok

**Comment by yolpsoftware** [comment]
@mxm87 can you please post the answer here? This issue appears on Google when people search for this.

**Reply by mxm87** [reply]
> @mxm87 can you please post the answer here? This issue appears on Google when people search for this.

Sorry for the late answer. The question was originally asked because there was a slight delay in the initial render if _initialScrollIndex_ was set to the middle of the list. But it turned out that this doesn't related to Reanimated.

This was because FlatList renders items in batches. So setting _initialNumToRender_ higher solved the issue in my case.
